applications/RomApplication/custom_utilities/rom_residuals_utility.h

```diff
@@ -138,17 +140,183 @@ namespace Kratos
         return MatrixResiduals;
         }
 
+        Matrix GetProjectedResidualsOntoPsi()
+        {
+            // Getting the number of elements and conditions from the model
+            const int nelements = static_cast<int>(mpModelPart.Elements().size());
+            const int nconditions = static_cast<int>(mpModelPart.Conditions().size());
+
+            const auto& CurrentProcessInfo = mpModelPart.GetProcessInfo();
+            const auto el_begin = mpModelPart.ElementsBegin();
+            const auto cond_begin = mpModelPart.ConditionsBegin();
+
+            //contributions to the system
+            Matrix LHS_Contribution = ZeroMatrix(0, 0);
+            Vector RHS_Contribution = ZeroVector(0);
+
+            //vector containing the localization in the system of the different terms
+            Element::EquationIdVectorType EquationId;
+            Matrix MatrixResiduals( (nelements + nconditions), mPetrovGalerkinRomDofs); // Matrix of reduced residuals.
+            Matrix PsiElemental;
+            #pragma omp parallel firstprivate(nelements, nconditions, LHS_Contribution, RHS_Contribution, EquationId, PsiElemental, el_begin, cond_begin)
+            {
+                #pragma omp for nowait
+                for (int k = 0; k < nelements; k++){
+                    auto it_el = el_begin + k;
+                    //detect if the element is active or not. If the user did not make any choice the element is active by default
+                    bool element_is_active = true;
+                    if ((it_el)->IsDefined(ACTIVE))
+                        element_is_active = (it_el)->Is(ACTIVE);
```

Comment:
```suggestion
                    const bool element_is_active = it_el->IsDefined(ACTIVE) ? it_el->Is(ACTIVE) : true;
```
On top of being less verbose, using the ternary operator allows you to make the variable  `const`.